charts/fluid-datamigrate/juicefs/templates/statefulset.yaml

```diff
@@ -0,0 +1,65 @@
+{{- if gt (.Values.datamigrate.parallelism | int) 1 }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ printf "%s-workers" .Release.Name }}
+  labels:
+    release: {{ .Release.Name }}
+    role: datamigrate-job
+  ownerReferences:
+  {{- if .Values.owner.enabled }}
+    - apiVersion: {{ .Values.owner.apiVersion }}
+      blockOwnerDeletion: {{ .Values.owner.blockOwnerDeletion }}
+      controller: {{ .Values.owner.controller }}
+      kind: {{ .Values.owner.kind }}
+      name: {{ .Values.owner.name }}
+      uid: {{ .Values.owner.uid }}
+  {{- end }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ printf "%s-workers" .Release.Name }}
+  # match the service name
+  serviceName: {{ printf "%s-svc" .Release.Name }}
+  replicas: {{ .Values.datamigrate.parallelism }}
```

Comment:
Can we scale in the data migrate workers when the migrate job is done? Correspondingly, we may want the workers scaling out when a cron datamigrate job is re-triggered.

Reply:
yes, I will add another commit to fix it.